packages/react/src/components/BarChartCard/BarChartCard.story.jsx

```diff
@@ -19,7 +19,7 @@ const sizes = Object.keys(CARD_SIZES).filter(
 const layouts = Object.keys(BAR_CHART_LAYOUTS);
 
 export default {
-  title: __DEV__ ? '1 - Watson IoT/⚠️ BarChartCard' : '1 - Watson IoT/BarChartCard',
+  title: __DEV__ ? '1 - Watson IoT/Card/⚠️ BarChartCard' : '1 - Watson IoT/Card/BarChartCard',
```

Comment:
@kevinsperrine @bjornalm Our `BarChartCard` test have been brought up to past our threshold, correct?

Reply:
Yes, technically I think all items that aren't deprecated have been. The only exceptions we have to the 90% rule are those in react/package.json: 

```json
    "exclude": [
      "src/**/*.story.js?(x)",
      "src/**/hooks/*.js",
      "src/components/SuiteHeader/util/suiteHeaderData.js",
      "src/components/SuiteHeader/util/uiresources.js",
      "src/components/FileUploader/stories/*.jsx",
      "src/components/Table/AsyncTable/*.js?(x)",
      "src/components/WizardInline/**/*.js?(x)",
      "src/components/Page/PageHero.jsx",
      "src/components/Page/EditPage.jsx",
      "src/components/Dashboard/Dashboard.jsx",
      "src/components/Dashboard/CardRenderer.jsx",
      "src/components/MapCard/storyFiles/*.jsx",
      "src/components/Table/Table.test.helpers.jsx",
      "src/components/Table/Table.story.helpers.jsx",
      "src/components/Table/TablePropTypes.js",
      "src/components/Table/TableColumnCustomizationModal/tableColumnCustomizationTestUtils.js"
    ],
```